documents/building/README.md

```diff
@@ -40,16 +40,62 @@ git clone --single-branch --branch master https://github.com/apache/celix.git
 ```
 
 ## Building and installing
-Apache Celix uses [CMake](https://cmake.org) as build system. CMake can generate (among others) makefiles.
+Apache Celix can be build using [Conan](https://conan.io) as package manager/build system or by directly using
+[CMake](https://cmake.org).
 
-### Building and installing with preinstalled libraries
+### Building Apache Celix using Conan
 The following packages (libraries + headers) should be installed on your system:
 
 * Development Environment
     * build-essentials (gcc/g++ or clang/clang++) 
     * java or zip (for packaging bundles)
     * make (3.14 or higher)
     * git
+    * cmake (3.18 or higher)
+    * Conan (2 or higher)
+
+For Ubuntu 20.04, use the following commands:
```

Comment:
I just checked `linux-build-apt` CI build, which uses Ubuntu 22.04.

Reply:
Correct and conan-create.yml also uses 22.04. The linux-build-conan in ubuntu still uses 20.04. 

I will update the documentation to 22.04.